routes/schedule_route.py

```
@@ -1,56 +1,40 @@
+from utils.jwt_utils import verify_token
+
+schedule_router = APIRouter(prefix="/schedule", tags=["schedule"])
 
-schedule_router = APIRouter()
 
-@schedule_router.get("/{user_id}", response_model=List[ScheduleResponseSchema])
-def get_schedule(user_id: int, db: Session = Depends(get_db)):
+@schedule_router.get("/", response_model=List[ScheduleResponseSchema])
+def get_schedule(
+    user_id: str = Depends(verify_token),
+    db: Session = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)